db/structure.sql

```diff
@@ -0,0 +1,148 @@
+--
+-- PostgreSQL database dump
```

Comment:
Let's rebase this now that #8 is merged and ensure the device/device_seq story is sane.